todo_app/Selenium Tests/test_Selenium.py

```diff
@@ -0,0 +1,47 @@
+import os, pytest, dotenv
+from threading import Thread
+from todo_app.data.trello_items import create_trello_board, delete_trello_board
+from todo_app import app
+from selenium import webdriver 
+from selenium.webdriver.support.ui import Select
+
+
+@pytest.fixture(scope='module')
+def app_with_temp_board():
+    board_id = create_trello_board('Test_Board')
```

Comment:
Here you're creating the board (which relies on your API key/token) before you've loaded the environment variables - this has issues when running the tests from the terminal

Reply:
This has now been moved after the .env is loaded

file_path = dotenv.find_dotenv('.env')
    dotenv.load_dotenv(file_path, override=True)
    
    board_id = create_trello_board('Test_Board')
    os.environ['TRELLO_API_BOARD_ID'] = board_id